### ISSUE TITLE
ERROR in vendor.c8030bb7976be5202ebc.js from UglifyJs

### ISSUE THREAD

**Issue by fumblehool** [webpack-contrib/uglifyjs-webpack-plugin#363]
* Operating System: Debian GNU/Linux 8.8 (jessie)
* Node Version: v7.10.1
* NPM Version: 4.2.0
* webpack Version: 3.3.0
* uglifyjs-webpack-plugin Version: 1.1.4

### Expected Behavior
On running `webpack --config config/webpack.prod.aot.js --progress --profile --bail` It should build the dist files

### Actual Behavior
Getting Error - 
```
ERROR in vendor.c8030bb7976be5202ebc.js from UglifyJs
TypeError: Cannot read property '_walk' of null
    at AST_Dot.eval (eval at <anonymous> (/app/angular/node_modules/uglifyjs-webpack-plugin/node_modules/uglify-es/tools/node.js:1:1), <anonymous>:1112:28)
```

**Comment by fumblehool** [comment]
@evilebottnawi Thanks for the quick reply! The issue was in `uglifyjs-webpack-plugin`.

**Comment by vtni** [comment]
How did you solved it?

**Comment by fumblehool** [comment]
@vtni I was trying to extend functionality of an older project. It used `uglifyjs-webpack-plugin@1.1.4`.
So I had to downgrade webpack to v3 and upgrade `uglifyjs-webpack-plugin@1.1.4` to `uglifyjs-webpack-plugin@1.3.0`
As pointed out by @evilebottnawi , problem was in uglify-js. This problem is fixed in `uglify-js@3.4.3`
